Nidhi.md

```diff
@@ -1,6 +1,5 @@
-# 29 Feb
 
->Brief:
+>Brief: The work done till 29 Feb
```

Comment:
I am saying there is no heading for this section.

Give it a heading like
`# till 29 Feb`